app/models/role.rb

```diff
@@ -128,7 +128,7 @@ def add_permissions(permissions, options = {})
     search = options.delete(:search)
 
     collection = Permission.where(:name => permissions).all
-    raise ArgumentError, 'some permissions were not found' if collection.size != permissions.size
+    raise ArgumentError, 'some permissions were not found' if collection.size < permissions.size
```

Comment:
@ares, can you  review please. In RHCI, 9 permission names yield 11 results from the DB.  (RHCI is adding 2 permissions with same names as permissions defined in foreman_discovery)